cmd/podman/generate_systemd.go

```diff
@@ -40,7 +39,10 @@ func init() {
 	containerSystemdCommand.SetHelpTemplate(HelpTemplate())
 	containerSystemdCommand.SetUsageTemplate(UsageTemplate())
 	flags := containerSystemdCommand.Flags()
-	flags.BoolVarP(&containerSystemdCommand.Name, "name", "n", false, "use the container name instead of ID")
+	flags.BoolVarP(&containerSystemdCommand.Name, "name", "n", false, "use the container/pod name instead of ID (deprecated nop)")
```

Comment:
Not a nop anymore :^) Had the impulse to do it. Will be fixed with the next revision.